tox.ini

```diff
@@ -19,14 +19,14 @@ commands =
 
 [testenv:check]
 deps =
-    black
     mypy
     ruff
+    setuptools
 skip_install = true
 commands =
     python setup.py check --strict --metadata
+    ruff format .
```

Comment:
Not sure if it is a good idea to change source files in CI or is ruff format only complaining if something isn’t formatted properly?

Reply:
ruff is complete drop-in for black. Idea was to decrease the dev dependencies and only use ruff for formatting and linting

Reply:
But good point. It need the --check option